network-providers/README.md

```diff
@@ -1,3 +1,15 @@
 # Network Providers
 
 This package holds components that are able to interact with the network (e.g. send transactions, query blockchain state), and their corresponding resources (e.g. request & responses structures).
+
+## Basic network providers
+
+- `BasicProxyNetworkProvider`, following the `IBasicNetworkProvider` interface - must be implemented by the SDK.
+- `BasicApiNetworkProvider`, following the `IBasicNetworkProvider` interface - must be implemented by the SDK.
+- `BasicNodeNetworkProvider`, following the `IBasicNetworkProvider` interface - must be implemented by the SDK.
+
+## Rich network providers
+
```

Comment:
maybe here could be Featured network provider

Reply:
or even better just NodeNetworkProvider

Reply:
Fixed.